### ISSUE TITLE
Custom highlight group for directories

### ISSUE THREAD

**Issue by smjonas** [danielfalk/smart-open.nvim#65]
Currently, the plugin uses the builtin highlight group `Directory` to highlight directories. Would you accept a PR that introduces a custom hlgroup such as `SmartOpenDirectory` (or any other name you might suggest) that would link to `Directory` by default? This would allow users to redefine the highlight only for the plugin, not globally, e.g. to an often lighter hlgroup such as `Comment .`

**Comment by danielfalk** [comment]
Yeah sure.  If it links to Directory by default and gives someone flexibility to highlight it differently that works.

**Comment by danielfalk** [comment]
This is in branch `0.3.x` now
